src/format/formats.ts

```diff
@@ -208,12 +261,15 @@ export function defaultWriterFormat(to: string): Format {
     writerFormat.pandoc.to = pandocTo;
   }
 
+  // Set the originating to
+  writerFormat.identifier[kTargetFormat] = to;
+
   // return the createFormat
   return writerFormat;
 }
 
 function powerpointFormat(): Format {
-  return createFormat("pptx", {
+  return createFormat("pptx", "Powerpoint", {
```

Comment:
@dragonstyle I believe this should be in the other order right ? 
This now creates file with Powerpoint extension... not good 😓 

Found that try to build a pptx with dev version, and it led me to this PR.

Reply:
I opened #3691 to fix this